Core/Rewriters/MockSetupRewriter.cs

```diff
@@ -45,7 +50,24 @@ public MockSetupRewriter (IFormatter formatter)
         baseCallNode = baseCallNode.WithAdditionalAnnotations (CreateAnnotation (originalNode, baseCallNode));
       }
 
-      return baseCallNode.WithLeadingAndTrailingTriviaOfNode (node);
+      return baseCallNode.WithLeadingTrivia (node.GetLeadingTrivia()).WithTrailingTrivia (SyntaxFactory.Whitespace (Environment.NewLine));
+    }
+
+    private InvocationExpressionSyntax TransformNode (InvocationExpressionSyntax invocationExpression)
```

Comment:
The method name should describe the intent, in what way does it transform the node?